app/code/Magento/Catalog/Block/Product/View/Attributes.php

```diff
@@ -85,8 +86,8 @@ public function getAdditionalData(array $excludeAttr = [])
                 } elseif ($attribute->getFrontendInput() == 'price' && is_string($value)) {
                     $value = $this->priceCurrency->convertAndFormat($value);
                 }
-
-                if (is_string($value) && strlen($value)) {
+    
+                if ($value instanceof Phrase || (is_string($value) && strlen($value))) {
```

Comment:
This failure to check for empty string results in empty attributes displaying as N/A in product view versus suppressing lines that have no value. Please fix to 2.1-develop !